packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -493,27 +508,38 @@ export default class ListControl extends React.Component {
       resolveWidget,
       parentIds,
       forID,
+      t,
     } = this.props;
 
     const { itemsCollapsed, keys } = this.state;
     const collapsed = itemsCollapsed[index];
     const key = keys[index];
     let field = this.props.field;
     const hasError = this.hasError(index);
-
-    if (this.getValueType() === valueTypes.MIXED) {
+    const isVariableTypesList = this.getValueType() === valueTypes.MIXED;
+    if (isVariableTypesList) {
       field = getTypedFieldForValue(field, item);
       if (!field) {
         return this.renderErroneousTypedItem(index, item);
       }
     }
-
     return (
       <SortableListItem
         css={[styles.listControlItem, collapsed && styles.listControlItemCollapsed]}
         index={index}
         key={key}
       >
+        {isVariableTypesList && (
+          <LabelComponent
+            field={field}
+            isActive={false}
+            hasErrors={hasError}
+            uniqueFieldId={this.uniqueFieldId}
+            isFieldOptional={field.get('required') === false}
```

Comment:
Moved this here since `field` is set here https://github.com/netlify/netlify-cms/pull/5544/files#diff-f01e2e8e89ff82812ef8ed9a8168d00752d696f8f6b5663f4f94c74312cbee86R521

Reply:
👍